.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "\U0001F41B Bug Report"
+about: Submit a bug report to help us improve Habitat
+
+---
+
+## 🐛 Bug
+
+<!-- A clear and concise description of what the bug is. -->
+
+## Command
+
+## To Reproduce
```

Comment:
Is this linebreak intentional or a typo?

Specifically, is it supposed to be:
## Command to Reproduce

as opposed to 

## Command 

## To Reproduce

Reply:
It's intentional. `Command` is for reflecting exact command to run the script. `To Reproduce` is for reflecting additional steps required to reproduce the bug.